### PR TITLE
Add cumulative forecasts

### PR DESCRIPTION
**Pull request by ttvand**

### CODE REVIEW

**Review by harahu — APPROVED** [review]

File: COVID_19_model.ipynb [hunk 1]
```diff
@@ -73,6 +73,7 @@
         "#@title Imports\n",
         "import contextlib\n",
         "import datetime\n",
+        "# from fbprophet import Prophet\n",
```
Comment (areeh) [inline COVID_19_model.ipynb:76]: Is this necessary?

**Review by areeh — COMMENTED** [review]
LGTM

File: COVID_19_model.ipynb (hunk 1 above)
Comment (ttvand) [reply COVID_19_model.ipynb:76]: No, I will drop it later

Comment (harahu) [reply COVID_19_model.ipynb:76]: Dropped in my PR #6

**Pull request merged**